tests/rptest/tests/cluster_config_test.py

```diff
@@ -1868,6 +1868,25 @@ def test_locked_configs(self):
         wait_for_version_sync(admin, self.redpanda,
                               patch_result['config_version'])
 
+        # Try to create a topic out-of-range of the locked min
+        rpk = RpkTool(self.redpanda)
+        try:
+            rpk.create_topic(
+                'pandatopic',
+                config={'segment.bytes': self.LOCKED_SEGMENT_SIZE_MIN - 1})
+        except RpkException as ex:
+            r = re.compile(
+                r'^.*STATUS\s+(?P<topic>\S+)\s+INVALID_CONFIG:(?P<errmsg>.+).$'
+            )
+            m = re.match(r, ex.msg)
+            if m is not None:
+                assert m.group('topic') == 'pandatopic'
+                errmsg = m.group('errmsg')
+                self.logger.debug(f'errmsg {errmsg}')
+                assert errmsg == 'segment.bytes is out of range of locked_min/max '
```

Comment:
note to readers: test currently fails here because the error message is incorrect. I'll figure that out Monday, I'm just letting you know why CI will fail.